src/views/homepage/components/pre-footer/pre-footer.module.css

```
@@ -0,0 +1,67 @@
+.preFooter {
+  --container-max-width: 1140px;
+  --grid-template-columns: 4;
+  --grid-column-gap: 24px;
+
+  position: relative;
+  padding: 64px 24px;
+  background-image: radial-gradient(
+    at bottom,
+    var(--token-color-palette-neutral-100),
+    --token-color-palette-neutral-0
```

I think the missing `var()` here is preventing the gradient from appearing:

```suggestion
    var(--token-color-palette-neutral-0)
```

---

ah, good catch!